cmd/tailscale/cli/metrics.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package cli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+)
+
+var metricsCmd = &ffcli.Command{
+	Name:      "metrics",
+	ShortHelp: "Show Tailscale metrics",
+	LongHelp: strings.TrimSpace(`
+
+The 'tailscale metrics' command shows Tailscale user-facing metrics (as opposed
+to internal metrics printed by 'tailscale debug metrics').
+
+For more information about Tailscale metrics, refer to
+https://tailscale.com/kb/1482/metrics
```

Comment:
This is a 404.

Also, normally we try to avoid linking to kb articles directly from the client. We link to `/s/foo` shortlink URLs instead:

```
bradfitz@book1pro tailscale.com % git grep tailscale.com/s/ | wc -l
      30
```

Reply:
The KB article is still in progress in https://github.com/tailscale/tailscale-www/pull/6863.

I've added a `/s/client-metrics` redirect to use here, but in practice it seems like we use kb links more often than short links:

```
$ git grep tailscale.com/kb/ '*.go' | wc -l
      34
$ git grep tailscale.com/s/ '*.go' | wc -l
      28
```

Reply:
I consider those regressions. :)

I've been bitten by KB reorgs enough (taking granular topics and merging them into a mega article, and then later resplitting it along different boundaries) that I instead pick the topic I care about in the client and deal with updating the redirects later as KB moves around.